### PR TITLE
Initial Demo

### PR DESCRIPTION
**Pull request by msr-lovely**

### CODE REVIEW

**Comment by jprester** [comment]
Can you use the color scheme as in the design?

For example, the blue button color should be #1D7EA2. The header background color should be: #000000 etc..

It would be also nice to see how you update the theme styles cause I am trying to figure it out.

File: package.json [hunk 1]
```diff
@@ -1,39 +1,20 @@
...
-    "react-dom": "^17.0.2",
-    "react-scripts": "4.0.3",
-    "web-vitals": "^1.0.1"
-  },
   "scripts": {
-    "start": "react-scripts start",
-    "build": "react-scripts build",
-    "test": "react-scripts test",
-    "eject": "react-scripts eject"
+    "dev": "next dev",
```
Comment (jprester) [inline package.json:6]: this is a minor thing but a lot of projects that people use to start with the default port of 3000, and in that case, the app won't run if something is already running there. Maybe you can just write it to sue different port like 3005

"dev": "next dev -p 3005"

**Comment by jprester** [comment]
And if possible can you add a header with logo like it is in design? Mostly so I can learn how to use this template approach.

File: pages/summaries/pick.js [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+    ]
+    const [chosenSummary, setChosenSummary] = useState('')
+    const [summaryIndex, setSummaryIndex] = useState(0)
+    const [confirmedChoices, setConfirmedChoices] = useState([])
+    const { getRootProps, getRadioProps } = useRadioGroup({
+        name: "summary",
+        onChange: setChosenSummary
+    })
+    const group = getRootProps()
+    const confirmChoise = () => {
```
Comment (jprester) [inline pages/summaries/pick.js]: change spelling confirmChoise => confirmChoice

File: package.json (hunk 1 above)
Comment (msr-lovely) [reply package.json:6]: This isn't going to be SSR (server side rendering) so no need for server port at all. dev is development, which is testing. When published on Netlify, there's no need to use ports.

The build command is npm run build, so npm run dev is actually not used on production :3

**Comment by msr-lovely** [comment]
> 
> 
> And if possible can you add a header with logo like it is in design? Mostly so I can learn how to use this template approach.

I added navigation bar with logo and title~

> 
> 
> Can you use the color scheme as in the design?
> 
> For example, the blue button color should be #1D7EA2. The header background color should be: #0 etc..
> 
> It would be also nice to see how you update the theme styles cause I am trying to figure it out.

So I changed the blue color to match the current theme, so the blue is #0bf

**Pull request merged**